corehq/ex-submodules/casexml/apps/case/tests/test_multimedia.py

```diff
@@ -300,3 +301,33 @@ def test_sync_log_invalidation_bug(self):
         # this used to fail before we fixed http://manage.dimagi.com/default.asp?158373
         self._doSubmitUpdateWithMultimedia(new_attachments=['commcare_logo_file'], removes=[],
                                            sync_token=sync_log._id)
+
+
+class CaseMultimediaS3DBTest(BaseCaseMultimediaTest):
+    """
+    Tests new attachments for cases and case properties
+    Spec: https://github.com/dimagi/commcare/wiki/CaseAttachmentAPI
+    """
+
+    def setUp(self):
+        super(CaseMultimediaS3DBTest, self).setUp()
+        with trap_extra_setup(AttributeError, msg="S3_BLOB_DB_SETTINGS not configured"):
+            config = settings.S3_BLOB_DB_SETTINGS
+            self.s3db = TemporaryS3BlobDB(config)
+            assert get_blob_db() is self.s3db, (get_blob_db(), self.s3db)
```

Comment:
nit: the above two lines don't need to (and probably shouldn't) be in the `with trap_extra_setup` block.